### PR TITLE
Add Cypress and add a first smoke test

### PR DESCRIPTION
**Pull request by dottorblaster**
As we need MOAR tests and we are moving towards a stabler product and a more intensive QA phase, I thought that putting together some smoke tests could be a good idea especially when it comes to identifying regressions.

I used Cypress to do this, I like _a lot_ this tool because it's easily scriptable and it can instrument both Chromium and Firefox.

### CODE REVIEW

**Review by fabriziosestito — APPROVED** [review]

File: test/e2e/cypress/integration/about.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+context('Trento About page', () => {
+  beforeEach(() => {
+    cy.visit('http://demo.trento-project.io');
+  });
```
Comment (fabriziosestito) [inline test/e2e/cypress/integration/about.js:4]: we could make this an env variable with a default so we can use it in our local forks/envs

**Comment by fabriziosestito** [comment]
Loving this!!

File: test/e2e/cypress/integration/about.js (hunk 1 above)
Comment (dottorblaster) [reply test/e2e/cypress/integration/about.js:4]: You're actually right, I have to make a tiny library to make the import and the setup of this kind of stuff easier

**Review by nelsonkopliku — APPROVED** [review]
yes-please
pg

**Comment by dottorblaster** [comment]
@fabriziosestito I added the `baseUrl` configuration, Cypress supports it out of the box. Also, we can override the baseUrl using the `CYPRESS_BASE_URL` env variable. What do you think?

**Pull request merged**